Assets/Camera/bevy_blendy_cameras.toml

```
@@ -0,0 +1,4 @@
+name = "bevy_blendy_cameras"
+description = "Blender inpspired editor/3Dview camera controllers and utilities"
```

```suggestion
description = "Blender inspired editor/3Dview camera controllers and utilities"
```
Typo.

(Optional) I'd also suggest `3D view` or `3D-view` instead of `3Dview`.